source/style-guide/terminology/general-term-guidelines/use-short-words.txt

```diff
@@ -4,13 +4,14 @@
 Use Short, Familiar Words and Phrases
 =====================================
 
-.. include:: /style-guide/includes/page-needs-update.rst
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
 
 Use short, familiar words and phrases to convey an idea clearly. Such
-words and phrases are more conversational, save space, are easier to
-scan, and are often easier for non-native English speakers to
-understand. Use a longer word or phrase only if necessary to convey a
-particular meaning.
+words and phrases are more conversational, save space, easier to scan,
```

Comment:
I think you need the "are" verb before "easier to scan" and "often easier...".

Reply:
Changed to list.